README.md

```diff
@@ -1,8 +1,46 @@
 # AWS Java Toolkit
 
-## Java for abstracting basic actions in AWS
+## A collection of small Java libraries for abstracting common operations on Amazon Web Services (AWS).
+
+### Requirements: 
+ 1. Java 17 or newer
+ 2. AWS Java SDK version 2
+ 3. Jackson (https://github.com/FasterXML/jackson)
+
+## Usage:
+The library is available in Maven Central. 
+Include it as a dependency in your project:  
+Example:
+```
+    <dependency>
+    <groupId>io.github.awsjavakit</groupId>
+    <artifactId>misc</artifactId>
+    <version>0.13.2</version>
+    </dependency>
+```   
+**Important Note:**   
+We are currently supporting only the specified versions of all the libraries that 
+aws-java-kit depends on. Different versions can produce a build error on the client's end.
+
+
+## Modules:
+All functionality is described in the test classes. Below you can find a short summary 
+of the implemented features.
+
+### Testing Utils (testingutils):
+**Description:**
+The `testingutils` module contains: 
+1. A set of simple fake clients for using during testing. 
+Most clients implement only a small subset of the actual actions.
+However, it is easy to add any missing functionality. 
```

Comment:
```
Most clients implement only a small subset of the actual actions.
However, it is easy to add any missing functionality.
```
-> 
```
Most clients implement only a limited subset of API methods, however, it is simple to extend the clients to add missing functionality.
```
The question is "how?"

Reply:
Good point about the "how".  I will add it later.